### PR TITLE
Some GUI polishing before pull request

### PR DESCRIPTION
**Pull request by marzsolt**

### CODE REVIEW

File: Client/Screen.py [hunk 1]
```diff
@@ -171,73 +153,102 @@ def onreturn_play_menu_input_ip(self, val):  # check onreturn of playMenu IP inp
...
+        if self.__screenState == sstatecons.ScreenState.MAIN_MENU:
+            if not self.__mainMenu.is_enabled():
+                self.__mainMenu.enable()
+            self.__mainMenu.mainloop(events)
+        elif self.__screenState == sstatecons.ScreenState.CONNECTION_MENU:
+            if not self.__connectionMenu.is_enabled():
+                self.__connectionMenu.enable()
+            self.__connectionMenu.mainloop(events)
+        elif self.__screenState == sstatecons.ScreenState.GAME:
+            self._game_screen()
```
Comment (DantElemer) [inline Client/Screen.py:235]: draw adequate screen function?

File: Client/Screen.py [hunk 2]
```diff
@@ -171,73 +153,102 @@ def onreturn_play_menu_input_ip(self, val):  # check onreturn of playMenu IP inp
...
+        for event in events:  # event handling - look at every event in the queue
+
+            if event.type == pg.KEYDOWN:  # did the user hit a key?
+
+                if event.key == pg.K_ESCAPE:  # Was it the Escape key? If so, stop the loop.
+                    running = False
+
+            # Did the user click the window close button? If so, stop the loop. w/o. doesn't work
+            if event.type == pg.QUIT:
+                running = False
```
Comment (DantElemer) [inline Client/Screen.py:247]: Outsourcing this into sg like a check_keyboard_exit_request function seems desirable for me :)

File: Client/Screen.py (hunk 1 above)
Comment (marzsolt) [reply Client/Screen.py:235]: excuse me :D ?

Comment (DantElemer) [reply Client/Screen.py:235]: Make a function for these lines and just call itfrom update

**Pull request merged**